pkg/rancher-desktop/main/deploymentProfiles.ts

```diff
@@ -169,6 +176,7 @@ function parseJsonFiles(rootPath: string, defaultsPath: string, lockedPath: stri
 class Win32DeploymentReader {
   protected registryPathProfile: string[];
   protected keyName = '';
+  protected errors: string[] = [];
```

Comment:
Please reset `errors` at the start of `readProfile`, so that we can be sure we don't end up with errors from a different run.

Still not a fan of keeping errors like this, but this is complicated enough that I don't want to bother with changing it right now.

Reply:
Each reader only is used once, but sure. 

Building the list of errors this way simplifies the recursive function calls.